bin/cron.js

```diff
@@ -157,7 +163,14 @@ cron.schedule("*/15 * * * *", async () => {
                   if (submission.score >= 90) {
```

Comment:
I missed this in the previous pr. The application quizzes don't have a score minimum on then so there doesn't need ot be a requirement for it to be above 90. Ideally, it should use the actual minimum stored in mongo but we can implement that later.

Reply:
I saw that yea that some badges just have "Application Quiz Attempted", wasn't sure. But I agree